jib-core/src/main/java/com/google/cloud/tools/jib/frontend/BuildStepsRunner.java

```diff
@@ -64,79 +94,74 @@ public static BuildImageStepsRunner newRunner(
     if (useOnlyProjectCache) {
       cachesInitializer.setBaseCacheDirectory(cacheDirectory);
     }
-
-    return new BuildImageStepsRunner(
-        buildConfiguration, sourceFilesConfiguration, cachesInitializer);
+    return cachesInitializer;
   }
 
-  private final Supplier<BuildImageSteps> buildImageStepsSupplier;
-
-  @VisibleForTesting
-  BuildImageStepsRunner(Supplier<BuildImageSteps> buildImageStepsSupplier) {
-    this.buildImageStepsSupplier = buildImageStepsSupplier;
+  /** @return true if Docker is installed on the user's system. */
+  public static boolean isDockerInstalled() {
```

Comment:
Move above private static method.

Reply:
Is there a style guide somewhere for the ordering? I keep forgetting it.